agent/base

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 4; sh-indentation: 4; tab-width: 8 -*-
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; sh-indentation: 4; tab-width: 8 -*-
```

Comment:
That's going to work if the file is edited with emacs (which is all I was thinking about when I made the comment), but it is not going to save us from the vims of this world...

Reply:
There is a `vi` based string we could add at the bottom to enforce it the same way, but people use other editors besides `vi` and `emacs`.

So, yes, this is a short term fix, and since `base` is going away when we switch to all Python3 in the future, this change seems enough for now.